thrust/thrust/iterator/detail/iterator_facade_category.h

```diff
@@ -132,16 +132,21 @@ struct iterator_facade_default_category_host
 template <typename Traversal, typename ValueParam, typename Reference>
 struct iterator_facade_default_category_device
     : thrust::detail::eval_if<
-        thrust::detail::is_convertible<Traversal, thrust::forward_traversal_tag>::value,
+        ::cuda::std::is_convertible<Traversal, thrust::forward_traversal_tag>::value,
         thrust::detail::eval_if<
-          thrust::detail::is_convertible<Traversal, thrust::random_access_traversal_tag>::value,
+          ::cuda::std::is_convertible<Traversal, thrust::random_access_traversal_tag>::value,
           thrust::detail::identity_<thrust::random_access_device_iterator_tag>,
-          thrust::detail::eval_if<thrust::detail::is_convertible<Traversal, thrust::bidirectional_traversal_tag>::value,
+          thrust::detail::eval_if<::cuda::std::is_convertible<Traversal, thrust::bidirectional_traversal_tag>::value,
                                   thrust::detail::identity_<thrust::bidirectional_device_iterator_tag>,
                                   thrust::detail::identity_<thrust::forward_device_iterator_tag>>>,
         thrust::detail::eval_if<
-          thrust::detail::is_convertible<Traversal, thrust::single_pass_traversal_tag>::value, // XXX note we differ
-                                                                                               // from Boost here
+          ::cuda::std::is_convertible<Traversal, thrust::single_pass_traversal_tag>::value, // XXX
+                                                                                            // note
+                                                                                            // we
+                                                                                            // differ
+                                                                                            // from
+                                                                                            // Boost
+                                                                                            // here
```

Comment:
Suggestion: reformat the comment into a single line and let clang-format break it again.